fbw-a32nx/src/systems/fmgc/src/efis/EfisSymbols.ts

```diff
@@ -352,7 +352,9 @@ export class EfisSymbols {
             }
 
             // TMPY
-            if (this.flightPlanService.hasTemporary) {
+            if (this.flightPlanService.hasTemporary && this.guidanceController.temporaryGeometry) {
+                const b = this.guidanceController.getGeometryForFlightPlan(FlightPlanIndex.Temporary);
+                console.log(b);
```

Comment:
```suggestion
```

Reply:
Sorry, don't know how that slipped through. Thanks for noticing.